src/ResourceManager/ContainerRegistry/Commands.ContainerRegistry/Commands.ContainerRegistry.csproj

```diff
@@ -0,0 +1,179 @@
+    <Reference Include="System.Data" />
+    <Reference Include="System.Xml" />
+    <Reference Include="System.Management.Automation, Version=3.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Commands\GetAzureContainerRegistry.cs" />
+    <Compile Include="Commands\TestAzureContainerRegistryNameAvailability.cs" />
+    <Compile Include="Commands\GetAzureContainerRegistryCredential.cs" />
+    <Compile Include="Commands\UpdateAzureContainerRegistry.cs" />
+    <Compile Include="Commands\RemoveAzureContainerRegistry.cs" />
+    <Compile Include="Commands\NewAzureContainerRegistry.cs" />
+    <Compile Include="Commands\UpdateAzureContainerRegistryCredential.cs" />
+    <Compile Include="Models\DeploymentTemplate.cs" />
+    <Compile Include="Models\PSContainerRegistry.cs" />
+    <Compile Include="Models\ContainerRegistryClient.cs" />
+    <Compile Include="Models\ContainerRegistryCmdletBase.cs" />
+    <Compile Include="Models\PSContainerRegistryCredential.cs" />
+    <Compile Include="Models\ResourceManagerClient.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <Content Include="Microsoft.Azure.Commands.ContainerRegistry.dll-Help.xml">
+      <SubType>Designer</SubType>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <Content Include="Microsoft.Azure.Commands.ContainerRegistry.format.ps1xml">
+      <SubType>Designer</SubType>
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </Content>
+    <None Include="..\AzureRM.ContainerRegistry.psd1">
+      <Link>AzureRM.ContainerRegistry.psd1</Link>
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
+    <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication\Commands.Common.Authentication.csproj">
+      <Project>{d3804b64-c0d3-48f8-82ec-1f632f833c9e}</Project>
+      <Name>Commands.Common.Authentication</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.ResourceManager.Common\Commands.ResourceManager.Common.csproj">
+      <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
+      <Name>Commands.ResourceManager.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Resources\Commands.ResourceManager\Cmdlets\Commands.Resources.Rest.csproj">
+      <Project>{8058d403-06e3-4bed-8924-d166ce303961}</Project>
+      <Name>Commands.Resources.Rest</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
```

Comment:
@djyou one additional note, you will need to add the following snippet to this file so the markdown files get copied over to the output directory: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Profile/Commands.Profile/Commands.Profile.csproj#L193-L198